base/generator.jl

```diff
@@ -20,6 +20,12 @@ function next(g::Generator, s)
     g.f(v), s2
 end
 
+"""
+    generate(f, c...) -> iterator
+
+Apply `f` to each element of `c`.
+For multiple collection arguments, apply f elementwise.
+"""
 generate(f, c...) = Generator(f, c...)
```

Comment:
@tkelman I agree with you, but in https://github.com/JuliaLang/julia/issues/10162 no clear decision was taken. I would have preferred to have only upper case constructors for all this family of iterators which are always of the same type. Maybe restart the discussion there?